CONTRIBUTING.md

```diff
@@ -0,0 +1,19 @@
+# Contribute
+
+## Where to start
+* Take a gander through the source code to familiarize yourself with the extractor.
+* If you're not sure where to start, here's a list of all currently known [issues](https://github.com/RichardLitt/extract-id3-tags-from-mp3s/issues),
+
+## Pull requests
+If you'd like to make a pull request, make sure to include the following:
+ - [ ] Description of the change you've made/added/introduced.
+ - [ ] Which issues, if any, you're addressing.
+
+## Reporting issues
+If you've found an issue and would like to submit it, do so [here](https://github.com/RichardLitt/extract-id3-tags-from-mp3s/issues/new)
+
+## Contact us
+Feel free to reach out to us! You can find us on [Slack](https://mntnr.slack.com/shared_invite/MTcxMDc5MTcxMjA1LTE0OTI1NDQ2OTQtYmMyZTc1N2Q2Ng).
```

Comment:
Please remove this line and the contact us section. I don't want to be on the hook for real time communication.